### ISSUE TITLE
Video module Issues

### ISSUE THREAD

**Issue by happycoder1337** [SocialEngine/phpv4-issues#108]
-Video modules does not detect proper URL.
We could not find a video there - please check the URL and try again. If you are sure that the URL is valid, please click here to continue.

-Thumbnail fetch not working.

**Comment by smazurov** [comment]
Could you please provide more info? What video are you attaching? If its youtube, have you followed the [Generate YouTube API key](http://support.socialengine.com/php/customer/portal/articles/2027828) instructions?

**Comment by happycoder1337** [comment]
yes, I've followed all the instructions, the video shows just doesnt detect its url and thumb

**Comment by smazurov** [comment]
Please see guidelines on how to submit [bug reports](https://github.com/SocialEngine/phpv4-issues/blob/master/CONTRIBUTING.md#bug-reports)

**Comment by MiamiShade** [comment]
Is this a confirmed bug on 4.9.0 ? or an older version

**Comment by smazurov** [comment]
No idea, we need more feedback

**Comment by happycoder1337** [comment]
this is on 4.8.9
I like github, because people can keep track of all the bugs, 
if we just use the standard bug report, we dont know what other bugs are there. 
Plus if bugs are reported here, it forces you guys to work on it sooner, you guys dont want too many people constantly reporting bugs. it makes you guys look bad. =P

**Comment by smazurov** [comment]
> Plus if bugs are reported here, it forces you guys to work on it sooner, you guys dont want too many 

I will reiterate that I cannot begin to attempt to help until a proper bug report is filed according to the guidelines I've linked. This isn't an attempt to dismiss this (or any other) bug report, but as it currently stands you have not provided enough information for us to determine if this is an issue with SE and needs to be addressed.

**Comment by happycoder1337** [comment]
I tested on a fresh copy of 4.8.9. what other information do you need?

**Comment by smazurov** [comment]
Please see guidelines on how to submit a proper [bug report](https://github.com/SocialEngine/phpv4-issues/blob/master/CONTRIBUTING.md#bug-reports). Here's the section more directly:

> **Isolate the problem** &mdash; ideally create a [reduced test case](http://css-tricks.com/6263-reduced-test-cases/) and a live example.

> A good bug report shouldn't leave others needing to chase you up for more information. Please try to be as detailed as possible in your report. What is your environment? What steps will reproduce the issue? What browser(s) and OS experience the problem? What would you expect to be the outcome? All these details will help people to fix any potential bugs.

**Comment by DonnaScriptTechs** [comment]
With Iframely added to 4.9.0, I cannot reproduce any issues regarding video URLs. We will close this as fixed but should you still have an issue after we release 4.9.0, please open a new ticket and please provide an example of an URL that doesn't work. Thank you.

**Comment by Elshara** [comment]
Awesome!
